parsers/JP.py

```diff
@@ -59,7 +59,10 @@ def fetch_production(zone_key='JP-TK', session=None, target_datetime=None,
                 'geothermal': None,
                 'unknown': df.loc[i, 'unknown']
             },
-            'storage': None,
+            'storage': {
```

Comment:
actually here it's better to just remove the storage key.
I will update the backend to not trigger an error on this.

Reply:
will do!